autopush/websocket.py

```diff
@@ -1223,9 +1224,9 @@ def check_missed_notifications(self, results, resume=False):
         if self.ps._check_notifications or self.ps._more_notifications:
             self.process_notifications()
 
-    def bad_message(self, typ, message=None):
+    def bad_message(self, typ, message=None, url=DEFAULT_ERR_URL):
         """Error helper for sending a 401 status back"""
-        msg = {"messageType": typ, "status": 401}
+        msg = {"messageType": typ, "status": 401, "more_info": url}
```

Comment:
Well, the problem is that we only really talk about error codes there. I'm wondering if it might make sense to break that bit out into it's own document.